src/functions.php

```diff
@@ -25,5 +25,5 @@ function getDataArray(object $obj): array
  */
 function getMigrationsPath(): string
 {
-    return __DIR__ . '\..\tests\lib\Migrations';
+    return __DIR__ . DIRECTORY_SEPARATOR . '..' . '/tests/lib/Migrations';
```

Comment:
If you want to support windows, you need this:

    return __DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR . 'tests' . DIRECTORY_SEPARATOR . 'lib' . DIRECTORY_SEPARATOR . 'Migrations';


Also please consider using `realpath()` function to eliminate `..` part.